.travis.yml

```diff
@@ -22,4 +22,4 @@ script:
      export PATH=$(npm bin):$PATH &&
      python3 setup.py install &&
      cd ../coala &&
-     coala --ci"
+     coala --ci';
```

Comment:
@aashutoshrathi Hey :) This step has indeed nothing to do with a `.coafile` for this repo itself. It's not even available in the running container. So revert those changes please